### ISSUE TITLE
✅ 🐞 Bug Report: New Menu Element: Hover Setting Not Working on Touch-Enabled Devices, Dropdown Opens on Click Instead - [ED-10688]

### ISSUE THREAD

**Issue by Daveden2** [elementor/elementor#22258]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

With the new menu widget, when the dropdown effect is set to **Open on Hover**, it works just fine on non-touchscreen PCs. But on touch-enabled devices, the dropdown menu opens on click rather than on hover regardless of which option is chosen.

### Steps to reproduce

1. Activate the Menu Experiment under _Elementor > Settings > Features > Menu_
2. Create a new header template in the Elementor Theme Builder
3. Drop in the new **Menu** Element to a container
4. Populate with menu items
5. Under the _Content > Layout_ tab, for one of the menu items, toggle on the **Dropdown Content** 
6. Under the **Dropdown Effect**, set to **Open on Hover** (which is the default). 
![image](https://user-images.githubusercontent.com/105490669/235434161-fd783f73-438a-474d-b66d-a590eab99e87.png)

7. Check on a non-touchscreen device (It should open on hover as expected)
8. Then check on a touch-enabled device (it would only open on click).

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.5.19
	PHP Version: 8.0.24
	PHP Memory Limit: 2048M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 2048M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://staging24.daveden.co.uk
	Home URL: https://staging24.daveden.co.uk
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 40M
	Max Memory limit: 2048M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/112.0

== Active Plugins ==
	All-in-One WP Migration
		Version: 7.73
		Author: ServMask

	Elementor
		Version: 3.12.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.3
		Author: Elementor.com

	HappyFiles Pro
		Version: 1.8.2
		Author: Codeer

	Hostinger
		Version: 1.3.0
		Author: Hostinger

	iThemes Security
		Version: 8.1.6
		Author: iThemes

	JetEngine
		Version: 3.1.4
		Author: Crocoblock

	LiteSpeed Cache
		Version: 5.4
		Author: LiteSpeed Technologies

	Performance Lab
		Version: 2.2.0
		Author: WordPress Performance Team


== Elements Usage ==
	
	footer : 2
		container : 6
		heading : 4
		html : 5
	header : 2
		button : 2
		container : 28
		heading : 1
		html : 6
		icon : 3
		icon-list : 1
		image : 1
		image-box : 1
		mega-menu : 1
		nav-menu : 1
		search-form : 1
		social-icons : 1
		text-editor : 1
		theme-site-logo : 1
	wp-post : 0
		accordion : 1
		blockquote : 3
		button : 23
		call-to-action : 1
		container : 101
		dce-tokens : 1
		divider : 3
		form : 1
		gallery : 1
		heading : 26
		html : 3
		icon-box : 1
		icon-list : 6
		image : 24
		image-box : 30
		image-carousel : 1
		image-gallery : 1
		jet-listing-grid : 1
		loop-grid : 2
		nav-menu : 1
		price-table : 3
		share-buttons : 1
		social-icons : 2
		text-editor : 17
	wp-page : 3
		container : 3
	loop-item : 2
		container : 2
		heading : 3
		image-box : 1
		jet-listing-dynamic-image : 1
	jet-listing-items : 0
		container : 1
		image-box : 1


== Settings ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 7
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Active

````

</details>

**Comment by nicholaszein** [comment]
Hi David! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hi @Daveden2!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.13.2**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
